draft-ietf-quic-multipath.md

```diff
@@ -351,31 +314,40 @@ does not discuss when a client decides to initiate a new path. We
 delegate such discussion in separate documents.
 
 To let the peer open a new path, an endpoint needs to provide its peer with connection IDs
-and associated Path Identifiers for the new path.
+for at least one unused path identifier. Endpoints SHOULD use the MP_NEW_CONNECTION_ID
+frame to provide new connection IDs and, respectively, the MP_RETIRE_CONNECTION_ID frame to
+retire connection IDs after a successful handshake indicating multipath support by both endpoints.
+
+Endpoints MUST NOT issue connection IDs with path identifiers larger than
+the path limitation declared by the initial_max_paths transport parameter
+or MAX_PATHS frames.
 
-To open a new path, an endpoint SHALL use different Connection IDs on different paths.
-Still, the receiver may observe the same Connection ID used on different
-4-tuples due to, e.g., NAT rebinding. In such case, the receiver reacts
+To open a new path, an endpoint MUST use a connection ID assciated with
```

Comment:
A typo. Also, given that we restrict now to only client-initiated paths, we could replace "endpoint" by "client". Then, we could specify that the server can reply to such path opening by sending packets with the newly seen Path ID on the perceived network path.

```suggestion
To open a new path, an endpoint MUST use a connection ID associated with
```